internal/conf/configuration.go

```diff
@@ -210,6 +210,7 @@ type ProviderConfiguration struct {
 	RedirectURL             string                     `json:"redirect_url"`
 	AllowedIdTokenIssuers   []string                   `json:"allowed_id_token_issuers" split_words:"true"`
 	FlowStateExpiryDuration time.Duration              `json:"flow_state_expiry_duration" split_words:"true"`
+	SkipNonceCheck          bool                       `json:"skip_nonce_check" split_words:"true"`
```

Comment:
should this be set at the provider level instead so the developer is able to skip the nonce check for google but enforce it for apple?